plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2547,11 +2547,23 @@ protected void setQuotaAndPeriod(VirtualMachineTO vmTO, CpuTuneDef ctd) {
     protected void enlightenWindowsVm(VirtualMachineTO vmTO, FeaturesDef features) {
         if (vmTO.getOs().contains("Windows PV")) {
             // If OS is Windows PV, then enable the features. Features supported on Windows 2008 and later
+            // https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10/html/configuring_and_managing_windows_virtual_machines/optimizing-windows-virtual-machines#enabling-hyper-v-enlightenments
             LibvirtVMDef.HyperVEnlightenmentFeatureDef hyv = new LibvirtVMDef.HyperVEnlightenmentFeatureDef();
             hyv.setFeature("relaxed", true);
             hyv.setFeature("vapic", true);
             hyv.setFeature("spinlocks", true);
-            hyv.setRetries(8096);
+            hyv.setRetries(8191);
+            hyv.setFeature("vendor_id", true);
+            hyv.setFeature("vpindex", true);
+            hyv.setFeature("runtime", true);
+            hyv.setFeature("synic", true);
+            hyv.setFeature("frequencies", true);
+            hyv.setFeature("reset", true);
+            hyv.setFeature("tlbflush", true);
+            hyv.setFeature("reenlightenment", true);
+            hyv.setFeature("stimer", true);
+            hyv.setFeature("ipi", true);
+            hyv.setFeature("evmcs", true);
```

Comment:
As per the docs "This feature is exclusive to Intel processors.". So, this could potentially cause issues with non-intel processors.

Reply:
hyperv on non-intel-like processors is maybe not so big an issue. We can
1. surround with extra code, 
2. add docs
3. never mind that possibility
4. drop this PR
I am leaning to the higher numbers, here.

Reply:
This is not about hyper-v, this is about hyper-v emulation on KVM itself. This PR isn't necessary given Vishesh's comments.